src/pages/errors/no-service-worker.tsx

```diff
@@ -0,0 +1,24 @@
+/**
+ * Page to display a user friendly message when `navigator.serviceWorker` is not available.
+ */
+
+import React from 'react'
+import About from '../../components/About.jsx'
+import Header from '../../components/Header.jsx'
+import '../default-page-styles.css'
+
+export default function NoServiceWorkerErrorPage (): React.JSX.Element {
+  return (
+    <>
+      <Header />
+      <main className='pa4-l bg-red-muted mw7 mt1 mb5 center pa4'>
```

Comment:
Cleaner without the thin white gap above
```suggestion
      <main className='pa4-l bg-red-muted mw7 mb5 center pa4'>
```